src/widgets/tdarr/Filename.tsx

```diff
@@ -0,0 +1,22 @@
+import { Text } from '@mantine/core';
+
+interface FilenameProps {
+  filename: string;
+}
+
+export function Filename(props: FilenameProps) {
+  const { filename } = props;
+
+  return (
+    <Text
+      style={{
+        whiteSpace: 'nowrap',
+        overflow: 'hidden',
+        textOverflow: 'ellipsis',
+      }}
+      size="xs"
+    >
+      {filename.substring(filename.lastIndexOf('/') + 1)}
```

Comment:
```TS 
filename.split('\\').pop().split('/').pop()
```
Strangely, it's the fastest in execution time and also works with every OS pathing (Which your way doesn't).